DigitalOffice/Services/UserService/Commands/UserLoginCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using LT.DigitalOffice.UserService.Repositories.Interfaces;
+using LT.DigitalOffice.UserService.Database.Entities;
+using LT.DigitalOffice.UserService.Commands.Interfaces;
+
+namespace LT.DigitalOffice.UserService.Commands
+{
+    public class UserLoginCommand : IUserLoginCommand
+    {
+        private readonly IUserRepository repository;
+
+        public UserLoginCommand([FromServices] IUserRepository repository)
+        {
+            this.repository = repository;
+        }
+
+        public DbUser Execute(string userEmail)
+        {
+            return repository.GetUserByEmail(userEmail); ;
+        }
+    }
+}
```

Comment:
Well, I think this class is redunant, because it just redirect result from repository. I do not think that it is required to isolate the repository into some command

Reply:
A design pattern is used to predetermine situations when business logic in the future will begin to grow and clog the code